### PR TITLE
add support for the SQL HAVING parameter in queries

### PR DESCRIPTION
**Pull request by robUx4**
so that all parameters are available

### CODE REVIEW

File: src/main/resources/org/jraf/androidcontentprovidergenerator/abstractselection.ftl [hunk 1]
```diff
@@ -34,9 +34,10 @@ public abstract class AbstractSelection<T extends AbstractSelection<?>> {
     private final StringBuilder mSelection = new StringBuilder();
     private final List<String> mSelectionArgs = new ArrayList<String>(5);
 
-    private Boolean mNotify;
-    private String mGroupBy;
-    private Integer mLimit;
+    Boolean mNotify;
```
Comment (BoD) [inline src/main/resources/org/jraf/androidcontentprovidergenerator/abstractselection.ftl]: Why did you remove "private"?

Comment (robUx4) [inline src/main/resources/org/jraf/androidcontentprovidergenerator/abstractselection.ftl]: Because I access the value in other classes I generate for that package. You can replace with getters if you want.

**Comment by BoD** [comment]
Thank you for this quality contribution.
